vault/resource_kmip_secret_backend.go

```diff
@@ -0,0 +1,223 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-provider-vault/util"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var kmipAPIFields = []string{
+	"default_tls_client_key_bits",
+	"default_tls_client_key_type",
+	"default_tls_client_ttl",
+	"listen_addrs",
+	"server_hostnames",
+	"server_ips",
+	"tls_ca_key_bits",
+	"tls_ca_key_type",
+	"tls_min_version",
+}
+
+func kmipSecretBackendResource() *schema.Resource {
+	return &schema.Resource{
+		Create: kmipSecretBackendCreate,
+		Read:   kmipSecretBackendRead,
+		Update: kmipSecretBackendUpdate,
+		Delete: kmipSecretBackendDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"path": {
+				Type:         schema.TypeString,
+				ForceNew:     true,
+				Optional:     true,
+				Default:      "kmip",
```

Comment:
What's the motivation for setting a default here?

Reply:
I tried to follow the `aws_secret_backend` [precedent here](https://github.com/hashicorp/terraform-provider-vault/blob/main/vault/resource_aws_secret_backend.go#L28). I noticed they use that default for documenting their import as well: https://github.com/hashicorp/terraform-provider-vault/blob/main/website/docs/r/aws_secret_backend.html.md#import

Reply:
Reverted to being required in 14d4add